### ISSUE TITLE
No git tags used for releases

### ISSUE THREAD

**Issue by davidffrench** [Apicurio/apicurio-operators#27]
## What
There are no git tags mapping to releases in this repo. This makes it difficult to include a specific version as a dependency in another project. I am using a commit hash to do this currently but I don't know which commit maps to a specific release.

**Comment by EricWittmann** [comment]
Good point.  @riprasad @jsenko  - we should update our upstream release procedures to also tag this repository whenever an Apicurito or Apicurio Registry release is performed.  We can add a tag associated with whatever project is being released.
